### PR TITLE
personal shield nerf

### PR DESCRIPTION
**Pull request by senorsapo**
Anyone that has faced them knows they can be abused to shit

1 TC cost means you can resist 5 shots per TC which is absurd

This is just what I think it should be, open to thoughts. Changes are in changelog for specifics.

### CODE REVIEW

**Comment by senorsapo** [comment]
!review

**Comment by 4000daniel1** [comment]
This might'd as well remove the item completely, I don't see any situation where using 8 TC, which is around half of an antagonists total TC pool, to block four projectile hits is worth it. Personally I'd say just increasing the price to 5 TC would be enough that people wouldn't buy several of them, which I believe is the main issue. A single one of them can be countered with a rapidly firing weapon just like how ballistics armor can be countered with lasers and w.e.

File: code/datums/uplink/devices and tools.dm [hunk 1]
```diff
@@ -71,7 +71,7 @@
 /datum/uplink_item/item/tools/personal_shield
 	name = "Personal Shield"
 	desc = "A personal shield that, when kept in your hand and activated, will protect its user from five projectile shots."
-	item_cost = 1
+	item_cost = 8
```
Comment (KingOfThePing) [inline code/datums/uplink/devices and tools.dm:74]: Way too much. This is more than half of a traitor's TC.
```suggestion
	item_cost = 4
```

File: code/game/objects/items/devices/personal_shield.dm [hunk 2]
```diff
@@ -6,10 +6,10 @@
 	item_state = "personal_shield"
 	contained_sprite = TRUE
 	slot_flags = SLOT_BELT
-	w_class = ITEMSIZE_LARGE
+	w_class = ITEMSIZE_HUGE
```
Comment (KingOfThePing) [inline code/game/objects/items/devices/personal_shield.dm:9]: Unnessecary, really.

**Review by KingOfThePing — CHANGES_REQUESTED** [review]
All in all - either increase the price reasonably OR nerf it's functionality. Both in one go makes this item utterly useless and there is no incentive to even buy it. At that point you can just remove it (don't do that).

**Comment by DreamySkrell** [comment]
> 1 TC cost means you can resist 5 shots per TC which is absurd

Ahelp any antag that buys more than one shield and cycles them. Cycling shields during combat is powergaming and bwoinkable.

Overall, this nerf is way too drastic and makes these shields 100% useless. See https://github.com/Aurorastation/Aurora.3/pull/14999 for an alternative that I have prepared.

**Comment by senorsapo** [comment]
Closing this in favor of Dreamy's PR

**Pull request closed without merging**